CHANGELOG.md

```diff
@@ -3,6 +3,15 @@
 The format is based on [Keep a Changelog](https://keepachangelog.com/en/1.0.0/)
 and this project adheres to [Semantic Versioning](https://semver.org/spec/v2.0.0.html).
 
+## [0.3.0] - 2024-08-21
```

Comment:
Could you also add the link to this tag like the other releases, even if the tag isn't created yet (in anticipation of when we release)?

Reply:
done here: [cf5248e](https://github.com/phac-nml/gasclustering/pull/28/commits/cf5248e00d7514936f2e1e0f96750577200c6c26)
😃